### ISSUE TITLE
Podman machine is running and shows in PD but no images shown

### ISSUE THREAD

**Issue by praveenkumar** [containers/podman-desktop#2428]
### Bug description

PD is showing that podman machine is running and same on the tray side

<img width="1151" alt="Screenshot 2023-05-10 at 5 59 39 PM" src="https://github.com/containers/podman-desktop/assets/410007/28449368-a9d3-4e9d-bc59-e42995e42e77">

But when check on the images it doesn't show any image even there are existing images which I can see using cli
```
podman images
REPOSITORY                        TAG         IMAGE ID      CREATED            SIZE
quay.io/praveenkumar/qotd         v1          4a9999a9b2e7  About an hour ago  220 MB
<none>                            <none>      76e6f94d68d4  2 hours ago        1.26 GB
quay.io/centos7/httpd-24-centos7  centos7     93b03918f23a  10 hours ago       356 MB
quay.io/podman/hello              latest      768f460ee981  11 hours ago       142 kB
mcr.microsoft.com/dotnet/sdk      6.0         a9ff2c65eb3a  7 days ago         768 MB
mcr.microsoft.com/dotnet/aspnet   6.0         56c76c4924c0  7 days ago         216 MB
```
<img width="1030" alt="Screenshot 2023-05-10 at 6 00 42 PM" src="https://github.com/containers/podman-desktop/assets/410007/67e93b14-8c4a-4904-9f86-d00d247ea5f4">

Even when try to pull the image it says no container runtime :(

<img width="1035" alt="Screenshot 2023-05-10 at 6 03 40 PM" src="https://github.com/containers/podman-desktop/assets/410007/a1cf3d07-6702-4fda-8760-937d77ac43e5">

### Operating system

13.3.1 (22E261) (M1)

### Version

0.15.0

**Comment by benoitf** [comment]
I would say it's a dup of https://github.com/containers/podman-desktop/issues/2417

**Comment by benoitf** [comment]
@praveenkumar 

Could you try the following

start the podman machine before starting Podman Desktop

like: `podman machine start`

and then launch Podman Desktop

are you facing the issue or it works using that flow ?

**Comment by benoitf** [comment]
closing as duplicate of https://github.com/containers/podman-desktop/issues/2417
